### ISSUE TITLE
[LE] Sorting background objects/tiles

### ISSUE THREAD

**Issue by Germanunkol** [michalove/platformer#142]
When selecting the background objects tool, a panel shows up. On that panel, all background objects (and individual tiles) should be accessible.
... but I have no idea how to sort them properly.
Currently, I've given categories (major and minor) to each tile. Same major category means the tiles will be on the same page, same minor category means it will try to place them next to each other.

However, I'm not happy at all. Issues:
- Some tiles should be next to each other, some below. Currently all tiles are placed in rows (horizontal) only. This breaks the doors, for example
- Some tiles only make sense inside a full object (trees, dragon). They should probably be removed.
- Full objects and individual tiles are currently mixed together. Maybe we should introduce tabs instead (buttons at the top allowing to switch between tiles, objects and categories or something?)
- Placing individual tiles is cumbersome - it's almost as bad as it was in Tiled. Maybe we should introduce "box-select and place" so that multiple tiles can be placed onto the map at once...?

So, bottom line is:
Do you have an idea how to sort them so that the user can quickly access and use tiles?
